cgmanifests/submodules/cgmanifest.json

```diff
@@ -219,22 +219,12 @@
       "component": {
         "type": "git",
         "git": {
-          "commitHash": "80d452484c5409444b0ec19383faa84bb7a4d351",
+          "commitHash": "59a2ac2745d8a57ac94c6accced73620d59fb844",
           "repositoryUrl": "https://github.com/pybind/pybind11.git"
         },
         "comments": "git submodule at cmake/external/onnx/third_party/pybind11"
       }
     },
-    {
```

Comment:
what was this component used for ?

Reply:
This change is because onnx updated their pybind submodule to a newer version.  And the newer one doesn't have this component anymore.  This cgmanifest.json file should be updated when we were updating the onnx submodule, but the author forgot this part.